### ISSUE TITLE
missing path to git?

### ISSUE THREAD

**Issue by aaminoff** [woahbase/alpine-ssh#1]
We are using this image in our CI/CD pipeline. Unfortunately, we think something in the recent commit changed the PATH or how bash is configured inside the image. We are seeing

![image](https://github.com/woahbase/alpine-ssh/assets/30444015/ae045507-646b-4945-88ea-27ebc03fc7d5)

It worked last week. If you can not fix this we will find another workaround, but thought you should be aware.

**Comment by lapnapra** [comment]
Yeah this is expected, the recent much-needed-overhaul-update removed `curl` and `git` APK files.  
If you need the old functionality back 

1. For the newer image with Openssh `9.6`, you can install the `git` APK at the startup of the container with the environment variable `S6_NEEDED_PACKAGES` (this will require internet connection available to the container). E.g.  `S6_NEEDED_PACKAGES="git"`.

2. Consider using the images tagged with [20210406](https://hub.docker.com/r/woahbase/alpine-ssh/tags?page=&page_size=&ordering=&name=20210406). These are the old images with Openssh `8.1`, and should work as expected.
